app/utils/constants.ts

```diff
@@ -81,3 +81,7 @@ export const studyFields = [
 ];
 
 export const starColor = '#7ed964';
+
+export const MAX_WINDOW_SIZE = 10000;
+
+export const DEFAULT_SEARCH_HASH = 'ANUr6Wwa';
```

Comment:
This default search hash might be handy later, but I haven't actually used it yet.